aws-greengrass-testing-platform/aws-greengrass-testing-platform-api/src/main/java/com/aws/greengrass/testing/platform/linux/LinuxPlatform.java

```diff
@@ -18,4 +19,9 @@ public LinuxPlatform(final Device device, final PillboxContext pillboxContext) {
     public LinuxCommands commands() {
         return new LinuxCommands(device, pillboxContext);
     }
+
+    @Override
+    public NetworkUtils getNetworkUtils() {
+        return new NetworkUtilsLinux();
```

Comment:
We could make this be a single instance there is nothing stateful about what `NetworkUtilsLinux` does. Let's built it in the constructor and just return it here